apps/metricstream/template.yaml

```diff
@@ -27,8 +28,8 @@ Parameters:
     Type: String
     Description: >-
       A file hosted in S3 containing list of metrics to stream.
-    Default: 's3://observeinc/cloudwatchmetrics/filters/recommended.yaml'
-    AllowedPattern: "^s3:\/\/.*"
+    Default: 's3://observeinc/cloudwatchmetrics/filters/empty.yaml'
+    AllowedPattern: "^(s3:\/\/.*)?$"
```

Comment:
why is the pattern changing?

Reply:
Since there are two options for how to configure metrics (via config file or via lambda), it is feasible that a user does not have a file to use. the first pattern forces a s3 file to be inserted. While we could have a parameter as a default, I think we should be flexible to someone entering nothing here (since we don't actually need this in the case of configuration via lambda).

Reply:
There's no way to specify that this param can be left blank? And the allowedPattern check still runs in that case?